app/services/katello/rhsm_fact_parser.rb

```diff
@@ -116,6 +116,11 @@ def cores
       facts['cpu.core(s)_per_socket']
     end
 
+    # Kernel version
```

Comment:
Nit: I'd drop this. It's exactly the same as the parent class and doesn't add anything.
```suggestion
```